config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
```

Comment:
this file will not be up to GIt, please
cp database.yml database.yml.example
git rm --cached config/database.yml
and in .gitignore add
/config/database.yml

Reply:
how to add into .gitignore?

Reply:
git add config/database.yml?

Reply:
open `.gitgnore` file, and write text `config/database.yml`

Reply:
open `.gitignore` and add `config/database.yml`
then run command `git rm --cached config/database.yml`